go/service/assets/api_op_CreateAsset_test.go

```diff
@@ -21,26 +20,31 @@ func Test_CreateAsset(t *testing.T) {
 		assert.NoError(t, err, "error while reading request body")
 		assert.JSONEq(t, `
 {
+	"@context": {
+		"edc": "https://w3id.org/edc/v0.0.1/ns/"
+	},
+	"@id": "",
 	"asset": {
-		"properties": {
-			"asset:prop:id": "1234",
-			"asset:prop:name": "product description",
-			"asset:prop:contenttype": "application/json"
+		"@id": "1234",
+		"edc:properties": {
+			"name": "product description",
+			"contentType": "application/json"
+		},
+		"edc:privateProperties": {
+			"private_name": "private private"
 		}
 	},
 	"dataAddress": {
-		"properties": {
-			"name": "Test asset",
-			"baseUrl": "https://jsonplaceholder.typicode.com/users",
-			"type": "HttpData"
-		}
+		"edc:type": "HttpData",
+		"edc:name": "Test asset",
+		"edc:baseUrl": "https://jsonplaceholder.typicode.com/users"
 	}
 }`, string(payload), "invalid payload")
 
 		fmt.Fprintf(w, `
 {
 	"createdAt": 1680004526,
```

Comment:
Since this is a response, it should be
```suggestion
	"edc:createdAt": 1680004526,
```